util/start_test

```diff
@@ -644,7 +645,7 @@ endif
 if ($?CHPL_TEST_UTIL_DIR) then
     set utildir = "$CHPL_TEST_UTIL_DIR"
 else
-    set utildir = "$CHPL_HOME/util"
+    set utildir = `dirname $0`
```

Comment:
I'm not sure if this is as easy as it seems.

Using release over release testing as an example. We try to the modern testing infrastructure and tests with the old compiler, runtime, and scripts to support the compiler. With the way utildir was set I think start_test would be using the chplenv scripts from the old releases (which you want), but now I think it will always use the ones located next to start_test